### DISCUSSION TITLE
URL Parameter für Einzelwerkzeuge möglich?

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#201] [Q&A]
Mit dem Parameter "tool" kann beim Aufruf einer Karte ein bestimmtes Werkzeug vorausgewählt werden.
Beispiel: tool=webgis.tools.editing.edit oder auch tool=webgis.tools.redlining.redlining. Dabei wird dann das entsprechende Menü mit weiteren Bearbeitungsoptionen dargestellt.
Ist es möglich eine noch weitere Vorauswahl vorzunehmen wie z.B: 
tool=webgis.tools.editing.edit.**insert**  oder auch 
tool=webgis.tools.redlining.redlining.**polygon** ?

Damit könnte man ein Set von parametrierten URLs für die Bearbeitung von Objekten für ganz bestimmte Aufgaben erstellen.

**Comment by fanie-mak** [comment]
Soweit ich weiß, geht es im WebGIS 5 nicht eine Vorauswahl bei einem Werkzeug zu machen. Zum Beispiel beim Editieren gleich mit dem neuen Objekt erfassen zu starten.

**Reply by faulgurke** [reply]
Schade.
Vielen Dank für die rasche Antwort!
